.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
         debug: [debug, no_debug]
         hup: [no_hup]
         module_type: ["static"]
+        path: [""]
```

Comment:
Can it be omitted if `path` is not required?

Reply:
Good catch.